dspace-api/src/test/data/dspaceFolder/config/local.cfg

```diff
@@ -129,3 +129,11 @@ configuration.exposed.array.value = public_value_1, public_value_2
 # Test config for the authentication ip functionality
 authentication-ip.Staff = 5.5.5.5
 authentication-ip.Student = 6.6.6.6
+
+# Test config for the usage statistics authorization
+usage-statistics.authorization.admin.usage = true
+usage-statistics.authorization.admin.workflow = true
+usage-statistics.authorization.admin.search = true
+
+# Test config for the versioning authorization
+versioning.item.history.view.admin = false
```

Comment:
These settings are all unnecessary, as they are the default values already in `versioning.cfg` and `usage-statistics.cfg`.

In any case, you also do NOT need to add most settings to this `local.cfg`.  Settings for Unit/Integration tests can be tweaked dynamically in the tests themselves by calling `configurationService.setProperty()` to change the value to whatever you want it to be.  I see you've done this already in some of your ITs, which is correct.

Configurations ONLY need to be added to this `local.cfg` if they CANNOT be reloaded/set dynamically.  There are very few configurations that fall into this category...mostly those that initialize our database/webapp.

In any case, all the above is just an FYI. These changes can be removed.

Reply:
@tdonohue Thank you for the feedback, I've removed the unnecessary config parameters in the test local.cfg file